roles/ceph-mon/tasks/create_mds_filesystems.yml

```diff
@@ -10,21 +10,30 @@
     - cephfs_metadata
   changed_when: false
 
+- name: check if ceph filesystem already exists
+  shell: ceph fs ls | sed -r "s/^name":" ([^,]+).*$/\1/" | egrep '^{{ cephfs }}$'
```

Comment:
I think it's easier to run: `ceph --cluster {{ cluster }} fs get {{ cephfs }}` and only create the fs if rc != 0